contents/docs/self-host/postgres-vs-clickhouse.md

```diff
@@ -4,27 +4,27 @@ sidebar: Docs
 showTitle: true
 ---
 
-## ClickHouse
+Long-term we think PostHog will have to run on [Clickhouse](https://clickhouse.tech) instead of Postgres as the main database to handle larger volumes of events and more robust analytics capabilities.
```

Comment:
```suggestion
We believe that PostHog will have to run on [Clickhouse](https://clickhouse.tech) instead of Postgres as the main database in the long term in order to handle larger volumes of events and provide more robust analytics capabilities.
```